templates/cocktail/home.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+
+    <h1>Type in your cocktail <img src="{{ asset('styles/osrs-cocktail.png') }}"
```

Comment:
Use new line for new elements, this way it will look more readable:
```suggestion
   

<h1>
    Type in your cocktail
    <img src="{{ asset('styles/osrs-cocktail.png') }}"/>
</h1>


```

Reply:
Also, don't write styling straight into template, add it separately in styling file (css)

Reply:
For that I have to install additional packages, which I will do that later. Used the dimension fixing for my own personal preference for now.

Reply:
okay, marking as resolved